src/transformers/models/align/processing_align.py

```diff
@@ -37,39 +37,26 @@
     import torch  # noqa: F401
 
 
-class AlignProcessorKwargs(ProcessingKwargs, total=False):
+class AlignProcessorKwargs(ProcessingKwargs, CommonKwargs, TextKwargs, ImagesKwargs, total=False):
     """
     Inherits from `ProcessingKwargs` to provide:
         1) Additional keys that this model requires to process inputs.
         2) Default values for extra keys.
     New keys have to be defined as follows to ensure type hinting is done correctly.
 
     ```python
-    common_kwargs: CommonKwargs = {
-            **CommonKwargs.__annotations__,
-        }
-        text_kwargs: TextKwargs = {
-            **TextKwargs.__annotations__,
-            "a_new_text_boolean_key": Optional[bool],
-        }
-        images_kwargs: ImagesKwargs = {
-            **ImagesKwargs.__annotations__,
-            "a_new_image_processing_key": Optional[int]
-        }
-    ```
+    images_kwargs: ImagesKwargs = {"new_image_kwarg": Optional[bool]}
 
-    """
-
-    common_kwargs: CommonKwargs = {
-        **CommonKwargs.__annotations__,
-    }
-    text_kwargs: TextKwargs = {
-        **TextKwargs.__annotations__,
-    }
-    images_kwargs: ImagesKwargs = {
-        **ImagesKwargs.__annotations__,
+    _defaults = {
+        "text_kwargs": {
+            "padding": "max_length",
+            "max_length": 64,
+        },
     }
 
+    ```
+    """
+
     _defaults = {
```

Comment:
```suggestion
    padding: "max_length"
    max_lenght: 64
```
should work no? Or does it not update the default for type-hints?

Reply:
yes it works for sure, this was to have a structured dict for defaults. Can change :)

Reply:
ah, now I remember, it actually can't work like that since [Typed Dicts don't support default values](https://github.com/python/mypy/issues/6131), they are made to hold the layout. They can have any attributes however, but it won't pass a value as default -like a `dataclass` would, but in this case we'd lose typing-, hence the manual operation

Reply:
ok got it thanks! Let's maybe comment about this!

Reply:
Do we have a comment for future code inspectors? I'm assuming here isn't the best place (we don't want it for all models) but didn't find a corresponding one elsewhere on a quick skim

Reply:
On that: there's doc in `processing_utils.ProcessingKwargs`, I added a comment nudging users to check there for documentation!